packages/wpcom-checkout/src/payment-methods/alipay.tsx

```diff
@@ -196,9 +196,11 @@ function isFormValid( store: AlipayStore ) {
 }
 
 function AlipayLabel() {
+	const { __ } = useI18n();
+
 	return (
 		<Fragment>
-			<span>Alipay</span>
```

Comment:
These names are intentionally not translated because they are proper nouns. That is, Alipay is always called "Alipay", even in French, Spanish, or Chinese.

Reply:
Ah, I guess that makes sense! But why would Apple Pay be translated, but Google Pay is not?

Reply:
Dropping these for now to unblock the style changes 👍